src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -273,7 +280,6 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
             if (installResult instanceof R2Error) {
                 this.$store.commit('error/handleError', installResult);
                 keepIterating = false;
-                this.isProfileBeingImported = false;
```

Comment:
Should we call `this.closeModal()` here? Or will that cause some issues?

Reply:
yeah, why not 👍